src/solvers/dgsem_p4est/containers_parallel.jl

```diff
@@ -184,6 +221,34 @@ function init_mpi_mortars!(mpi_mortars, mesh::ParallelP4estMesh, basis, elements
     return mpi_mortars
 end
 
+function Adapt.adapt_structure(to, mpi_mortars::P4estMPIMortarContainer)
+    # Only parts of this container are adapted, since we currently don't
+    # use `local_neighbor_ids`, `local_neighbor_positions`, `normal_directions`
+    # on the GPU. If we do need them we need to redesign this to use the VecOfArrays
+    # approach.
```

Comment:
```suggestion
    # TODO: GPU
    # Only parts of this container are adapted, since we currently don't
    # use `local_neighbor_ids`, `local_neighbor_positions`, `normal_directions`
    # on the GPU. If we do need them we need to redesign this to use the VecOfArrays
    # approach.
```